src/main/java/nextstep/ladder/domain/Line.java

```diff
@@ -0,0 +1,41 @@
+package nextstep.ladder.domain;
+
+import java.util.Objects;
+
+public class Line {
+
+    private final LineRange lineRange;
+    private boolean hasPoint;
+
+    public Line(boolean hasPoint, int left) {
+        this.hasPoint = hasPoint;
+        this.lineRange = new LineRange(left);
+    }
```

Comment:
```suggestion
    public Line(boolean hasPoint, int left) {
        this(hasPoint, new LineRange(left));
    }
```

주 생성자를 추가하고 주 생성자를 호출한다.